### PR TITLE
feat: add webhook reference doc

### PR DESCRIPTION
**Pull request by connor4312**
Should be mostly complete, feel free to leave comments or just push up any corrections/additions you may have :)

### CODE REVIEW

File: src/reference/constellation/events_table.pug [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+                        td
+                            div!= marked(data.description)
+                            if data.payload
+                                strong Payload:&nbsp;
+                                small
+                                    +typeNameString({ type: data.payload.type, required: true })
+                                +simpleType('', data.payload, true)
+                            if data.example
+                                p
+                                    a(href=`#example_${name.replace(/\W/g, '_')}`) View example
```
Comment (SimonSchick) [inline src/reference/constellation/events_table.pug]: Seen that a few times now, might want to consolidate this into a util.

File: src/reference/webhooks/verify.js.txt [hunk 2]
```diff
@@ -0,0 +1,8 @@
+const crypto = require('crypto');
```
Comment (SimonSchick) [inline src/reference/webhooks/verify.js.txt]: `.txt`?

Comment (connor4312) [reply src/reference/webhooks/verify.js.txt]: To keep eslint from linting it, comtpalining about unused functions and such

Comment (SimonSchick) [reply src/reference/webhooks/verify.js.txt]: You could exclude the file from eslint, which would be cleaner imo, but I will leave this up to you.

**Pull request merged**